app/main.py

```diff
@@ -1,34 +1,164 @@
+class IncorectDataInput(Exception):
+    pass
+
+
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
```

Comment:
Place all arguments in the `__init__` here and in the other classes on a new line to improve the readability.